app/src/reportWebVitals.ts

```diff
@@ -1,15 +1,15 @@
-import { ReportHandler } from 'web-vitals';
```

Comment:
Same comment as my other one, looks like your base branch is out of date. This appears to be undoing the prettified styling but you can easily get up to date using git fetch/git merge (full command in my other comment)